Session.xcodeproj/xcshareddata/xcschemes/Session.xcscheme

```diff
@@ -157,7 +157,7 @@
       </Testables>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "App Store Release"
```

Comment:
Is this an intentional change?

Reply:
I don't think so. I'll switch back to `Debug`